src/documents/consumer.py

```diff
@@ -578,6 +594,92 @@ def progress_callback(current_progress, max_progress):  # pragma: no cover
 
         return document
 
+    def get_template_overrides(
+        self,
+        input_doc: ConsumableDocument,
+    ) -> DocumentMetadataOverrides:
+        """
+        Match consumption templates to a document based on source and
+        file name filters, path filters or mail rule filter if specified
+        """
+        overrides = DocumentMetadataOverrides()
+        for template in ConsumptionTemplate.objects.all().order_by("order"):
+            template_overrides = DocumentMetadataOverrides()
+
+            if document_matches_template(input_doc, template):
+                if template.assign_title is not None:
+                    template_overrides.title = template.assign_title
+                if template.assign_tags is not None:
+                    template_overrides.tag_ids = [
+                        tag.pk for tag in template.assign_tags.all()
+                    ]
+                if template.assign_correspondent is not None:
+                    template_overrides.correspondent_id = (
+                        template.assign_correspondent.pk
+                    )
+                if template.assign_document_type is not None:
+                    template_overrides.document_type_id = (
+                        template.assign_document_type.pk
+                    )
+                if template.assign_storage_path is not None:
+                    template_overrides.storage_path_id = template.assign_storage_path.pk
+                if template.assign_owner is not None:
+                    template_overrides.owner_id = template.assign_owner.pk
+                if template.assign_view_users is not None:
+                    template_overrides.view_users = [
+                        user.pk for user in template.assign_view_users.all()
+                    ]
+                if template.assign_view_groups is not None:
+                    template_overrides.view_groups = [
+                        group.pk for group in template.assign_view_groups.all()
+                    ]
+                if template.assign_change_users is not None:
+                    template_overrides.change_users = [
+                        user.pk for user in template.assign_change_users.all()
+                    ]
+                if template.assign_change_groups is not None:
+                    template_overrides.change_groups = [
+                        group.pk for group in template.assign_change_groups.all()
+                    ]
+                overrides.update(template_overrides)
+        return overrides
+
+    def _parse_title_placeholders(self, title: str) -> str:
+        """
+        Consumption template title placeholders can only include items that are
+        assigned as part of this template (since auto-matching hasnt happened yet)
+        """
+        local_added = timezone.now()
```

Comment:
This won't actually be local time.  It will be an aware datetime in UTC.  You'll want to wrap it as `timezone.localdate(timezone.now())` to have the actual local time